src/platform/ipc_stream_test.c

```
@@ -319,6 +321,89 @@ test_ipc_listen_activation_bad_arg(void)
+	nng_aio_wait(laio);
+	NUTS_PASS(nng_aio_result(laio));
+
+	c1 = nng_aio_get_output(daio, 0);
+	c2 = nng_aio_get_output(laio, 0);
+
+	// 8 x (INT_MAX/8 + 1) = INT_MAX + 1: cumulative iov just clears the
+	// 32-bit signed kernel cap while committing only ~256 MiB of memory.
+	const size_t per = ((size_t) INT_MAX / 8) + 1;
+	char        *buf = malloc(per);
+	NUTS_ASSERT(buf != NULL);
+	memset(buf, 'A', per);
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)